QuotesApp/Core/BookQuotesScreen/QuotesListRowView.swift

```diff
@@ -10,62 +10,73 @@ import SwiftUI
 struct QuotesListRowView: View {
     private let quote: Domain.QuoteItem
     private let action: () -> Void
+    private let onDelete: () -> Void
+    @Binding private var activeRow: UUID?
     
-    init(quote: Domain.QuoteItem, action: @escaping () -> Void) {
+    init(quote: Domain.QuoteItem,
+         activeRow: Binding<UUID?>,
+         action: @escaping () -> Void,
+         onDelete: @escaping () -> Void) {
```

Comment:
```suggestion
    init(
         quote: Domain.QuoteItem,
         activeRow: Binding<UUID?>,
         action: @escaping () -> Void,
         onDelete: @escaping () -> Void
         ) {
```
To jest prawidłowy sposób rozdzielenia tego an linie.